scripts/get_config_map_status.sh

```diff
@@ -15,6 +15,8 @@ done
 
 if [[ $COUNTER -eq $MAX_ATTEMPTS ]]; then
   echo "ConfigMap '$CONFIGMAP_NAME' did not become available within $MAX_ATTEMPTS attempts."
+  #  Output for debugging
```

Comment:
probably want to remove this before merging?

Reply:
I think leave it, if this code gets hit the module will fail and it could provide useful debug info